python/langsmith/pytest_plugin.py

```diff
@@ -41,8 +41,8 @@ def _handle_output_args(args):
     """Handle output arguments."""
     if any(opt in args for opt in ["--langsmith-output"]):
         # Only add --quiet if it's not already there
-        if not any(a in args for a in ["-q", "--quiet"]):
```

Comment:
i think we still want to check for presence of -q or --quiet?

Reply:
Why?

Reply:
Seems to work fine: https://www.loom.com/share/764ecd32c495459db8d7095a3f8b4a80